### ISSUE TITLE
Error message in BioGeoBears RASP

### ISSUE THREAD

**Issue by Geroto** [sculab/RASP#6]
Hello
Any time I try run biogebears in RASP (compare models, on Trees or Consensus) the software returns  the follow error message:

Could not get BioGepBears result file! Check your data or setting, please!

And I don't know what I'm doing wrong. Any tips?
Thanks in advance.
Caio Geroto

**Comment by ymilesz** [comment]
I am having the same problem, were you able to figure this out?

**Comment by sculab** [comment]
> I am having the same problem, were you able to figure this out?

Could you complete the analysis with the example file?

**Comment by ymilesz** [comment]
No, same error. I think maybe biogeobears isn't installed when I click on the install 3rd party R packages.

**Comment by sculab** [comment]
> No, same error. I think maybe biogeobears isn't installed when I click on the install 3rd party R packages.

The current version of RASP has all of the necessary R packages within it, and does not need to install 3rd party R packages. Could you give me more information about your operating system and the and the installation path of the RASP?  I will try to reproduce the problem on my computer.

**Comment by ymilesz** [comment]
Okay so I moved the folder to my PC download folder and it worked, it was sitting in my OneDrive before that.

**Comment by Geroto** [comment]
Hum... try that solution in my pc and still not working... any other ideas?
